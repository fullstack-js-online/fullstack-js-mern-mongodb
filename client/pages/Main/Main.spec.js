import React from 'react'
import { shallow } from 'enzyme'

import { Main } from './'

describe('Main component', () => {
  const props = {
    isAuthenticated: false,
    flash: {
      messages: []
    },
    dispatch: jest.fn(),
    user: {},
    validationSchema: {},
    onSubmit: jest.fn()
  }

  it('should match snapshot', () => {
    expect(shallow(<Main {...props} />)).toMatchSnapshot()
  })
})
