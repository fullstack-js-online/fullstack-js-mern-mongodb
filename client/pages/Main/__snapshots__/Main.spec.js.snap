// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main component should match snapshot 1`] = `
<BrowserRouter>
  <div
    className="page pb-16"
  >
    <FlashMessages
      messages={Array []}
    />
    <SendEmailConfirm
      resendEmailConfirmation={[Function]}
    />
    <withRouter(Topbar)
      isAuthenticated={false}
      logout={[Function]}
      user={Object {}}
    />
    <Route
      component={[Function]}
      path="/"
    />
    <Route
      component={[Function]}
      path="/auth/login"
    />
    <Route
      component={[Function]}
      path="/auth/register"
    />
    <Route
      component={[Function]}
      path="/auth/passwords/reset/:token"
    />
    <Route
      component={[Function]}
      path="/auth/passwords/email"
    />
    <Route
      component={[Function]}
      path="/auth/emails/confirm/:token"
    />
  </div>
</BrowserRouter>
`;
